src/node_crypto_bio.cc

```diff
@@ -27,6 +28,21 @@ BIO* NodeBIO::New() {
 }
 
 
+BIO* NodeBIO::NewFixed(const char *data, size_t len) {
```

Comment:
Nit, but let's use `const char* data` here. This is how it is in the rest of the file.